apps/ledger-live-mobile/e2e/page/index.ts

```diff
@@ -105,10 +110,18 @@ export class Application {
     testAccounts,
   }: ApplicationOptions) {
     let proxyPort = 0;
+    const isLDMKEnabled = Boolean(
```

Comment:
[SHOULD] If you need to force a boolean, I'd rater you force it in the return type
eg:
```
return !!getFeature({ key: "ldmkTransport", localOverrides })?.enabled;
```